rskj-core/src/test/java/co/rsk/peg/BridgeUtilsTest.java

```diff
@@ -511,6 +515,79 @@ public void getCoinFromBigInteger() {
         Assert.assertEquals(Coin.COIN, BridgeUtils.getCoinFromBigInteger(BigInteger.valueOf(Coin.COIN.getValue())));
     }
 
+    @Test(expected = Exception.class)
+    public void validateHeightAndConfirmations_invalid_height() throws Exception {
+        BridgeUtils.validateHeightAndConfirmations(-1, 0, 0, null);
+    }
+
+    @Test
+    public void validateHeightAndConfirmation_insufficient_confirmations() throws Exception {
+        Assert.assertFalse(BridgeUtils.validateHeightAndConfirmations(2, 5, 10, Sha256Hash.of(Hex.decode("ab"))));
+    }
+
+    @Test
+    public void validateHeightAndConfirmation_enough_confirmations() throws Exception {
+        Assert.assertTrue(BridgeUtils.validateHeightAndConfirmations(2, 5, 3, Sha256Hash.of(Hex.decode("ab"))));
+    }
+
+    @Test(expected = Exception.class)
+    public void calculateMerkleRoot_invalid_pmt() {
+        BridgeUtils.calculateMerkleRoot(mock(NetworkParameters.class), Hex.decode("ab"), null);
```

Comment:
Be careful sending a mock in this way, plus expecting `Exception`. The method could be failing with an NPE if internally depends on any value from the mocked NetworkParameters.